test/e2e/test_job_submissions.py

```diff
@@ -3282,3 +3283,109 @@ def test_worker_job_attachments_output_only(
             reference_dir_path=f"{os.path.dirname(__file__)}/job_attachment_bundle/output_only_job/correct_output",
             output_dir_path=output_root_path + "/output",
         )
+
+    @pytest.mark.skipif(
+        os.environ["OPERATING_SYSTEM"] == "windows",
+        reason="Linux specific job bundle to test create job API call",
+    )
+    def test_worker_create_job_API_call_linux(
+        self,
+        deadline_resources: DeadlineResources,
+        deadline_client: DeadlineClient,
+        session_worker: EC2InstanceWorker,
+    ) -> None:
+        job_bundle_path: str = os.path.join(
+            os.path.dirname(__file__), "job_attachment_bundle", "complex_bundle", "linux"
+        )
```

Comment:
bit a of a nit that `complex` isn't very descriptive. Would like if we could capture this bit from your PR description in a doc string here so that future readers can quickly grasp what makes the bundle complex

> Add Linux and Windows tests that using CreateJob with a complex job bundle include:
> 
> Have more than one manifests and root path
> Have step-step dependency
> Have output
> And for Linux, test also setup storage profile